lib/schnorr.py

```diff
@@ -148,20 +151,24 @@ def sign(privkey, message_hash):
             # just leave it as a vague exception.
             raise ValueError('could not sign')
         return bytes(sig)
-    else:
+    else:      
         # pure python fallback:
         G = ecdsa.SECP256k1.generator
         order = G.order()
         fieldsize = G.curve().p()
 
+        # For pure python (not libsecp256k1), convert an empty ndata to bytes as the required format for concatenation inside the nonce function.
+        if ndata == None:
```

Comment:
Usually the pythonic  way is `if ndata is None:`  Even though what you have here will work and is equivalent.

Reply:
fixed in 35e621f